src/components/header/breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,37 @@
+/// <reference types="vite-plugin-svgr/client" />
+import React from "react";
+import ArrowDown from "@/assets/arrow-down.svg?react";
+import Divider from "@/assets/divider.svg?react";
+import Typography from "@/components/typography/Typography";
+
+type BreadcrumbProps = {
+  breadcrumb: BreadcrumbItem;
+};
+
+export type BreadcrumbItem = {
+  text: string;
+  isDropdown?: boolean;
+  shouldAddDivider?: boolean;
+};
+
+const Breadcrumb = React.memo(({ breadcrumb }: BreadcrumbProps) => (
```

Comment:
NIce `React.memo` 😍 
Probably better to use primitivse here to make sure no rerenders are happpening for no reason